### ISSUE TITLE
Add a setup.py and requirements.txt

### ISSUE THREAD

**Issue by pbui** [pbui/yldme#2]
Add a `setup.py` script that installs yldme and related files to a suitable system location.

- [x] Add `requirements.txt` that lists dependencies
- [ ] Add `setup.py` that install script and data support files in the proper location
- [ ] Modify `yldme.py` to search appropriate locations for support files

**Comment by noah-yoshida** [comment]
What are you referring to by 'support files'? The stuff in `assets/`?

**Comment by pbui** [comment]
Yes, both `assets` and `templates` need to be put somewhere so that `yldme.py` can find them at run-time.

I'm not sure what the best practice is for Python applications, but that is something we should lookup and follow.

**Comment by noah-yoshida** [comment]
So you want `setup.py` to not simply clone the repo then?

**Comment by noah-yoshida** [comment]
I think the standards are different for different frameworks - Django and Flask have different conventions iirc

**Comment by noah-yoshida** [comment]
https://github.com/pbui/yldme/pull/6

**Comment by pbui** [comment]
> So you want `setup.py` to not simply clone the repo then?

No, it should install yldme.py to something like `/usr/bin` and the support files to `/usr/share/`.

**Comment by noah-yoshida** [comment]
Is there any reason that `~/.config/yldme/db` was chosen as the db path? Would it make more sense to keep it within the project directory?

**Comment by pbui** [comment]
On Linux/Unix user settings generally should go in `~/.config/<project>/`.  This should be configurable though.

**Comment by noah-yoshida** [comment]
Would the db be considered a user setting though?

**Comment by pbui** [comment]
> Would the db be considered a user setting though?

Yes, its location would be a user setting.

**Comment by AndroidKitKat** [comment]
PyYaml is missing from requirements.txt
